src/WebJobs.Script/Utility.cs

```diff
@@ -996,6 +997,56 @@ public static FunctionAppContentEditingState GetFunctionAppContentEditingState(I
             }
         }
 
+        public static void GetScaleInstancesToProcess(
```

Comment:
It would be helpful to have xml doc for methods indicating what they're for. Otherwise, readers (like me) have to grok through the code to figure it all out.